packages/resolve-redux/src/send_command_middleware.js

```diff
@@ -18,11 +19,8 @@ export default params => store => next => (action) => {
                 payload
             })
             .catch((error) => {
-                const errorAction = Object.keys(action).reduce(
-                    (result, field) =>
-                        field === 'command' ? result : { ...result, [field]: action[field] },
-                    { status: 'error', error }
-                );
+                const errorAction = { ...action };
+                errorAction.command.error = error;
```

Comment:
Use
```
{
  ...action,
  command: {
    ...action.command,
    error
  }
}
```
Otherwise you will modify the 'command' object by the reference